README.rst

```diff
@@ -275,3 +289,12 @@ Step 6: save the template file
 .. |step6_2| image:: https://github.com/pibooth/pibooth-picture-template/blob/master/docs/images/step6_2_export.png?raw=true
    :width: 80 %
    :alt: step6_2_export
+
+Development
+-------------------
+
+
+::
+
+    $ pip3 install -r requirements.txt
```

Comment:
No, the way to do it in pibooth project is use the setup.py:

pip install -e .